resources/js/Components/EventCard.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+
+const props = withDefaults(
```

Comment:
You don't need to assign props to a variable because you're only using them in the `template` section.

```suggestion
withDefaults(
```